openwisp_controller/config/base/vpn.py

```diff
@@ -187,10 +203,35 @@ def _validate_subnet_ip(self):
                     {'ip': _('VPN IP address must be within the VPN subnet')}
                 )
 
+    def _validate_host_and_auth_token(self):
+        if self.host and self.auth_token and self._is_backend_type('zerotier'):
+            try:
+                response = ZerotierService(self.host, self.auth_token).get_node_status()
+            except ConnectionError as err:
+                err_msg = f'Failed to connect to the ZeroTier controller, error: {err}'
+                raise ValidationError({'host': _(err_msg)})
```

Comment:
From [Django's translation docs](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#formatting-strings-format-lazy:~:text=(The%20caveat%20with%20using%20variables%20or%20computed%20values%2C%20as%20in%20the%20previous%20two%20examples%2C%20is%20that%20Django%E2%80%99s%20translation%2Dstring%2Ddetecting%20utility%2C%20django%2Dadmin%20makemessages%2C%20won%E2%80%99t%20be%20able%20to%20find%20these%20strings.%20More%20on%20makemessages%20later.)) 

> (The caveat with using variables or computed values, as in the previous two examples, is that Django’s translation-string-detecting utility, [django-admin makemessages](https://docs.djangoproject.com/en/4.2/ref/django-admin/#django-admin-makemessages), won’t be able to find these strings. More on [makemessages](https://docs.djangoproject.com/en/4.2/ref/django-admin/#django-admin-makemessages) later.)

We need to update the error messages.